File: Vian Rechel Academy Incorporated (Multi-page App)/app/src/main/java/com/example/vianrechelacademyincorporatedmulti_pageapp/SplashActivity.java
package com.example.vianrechelacademyincorporatedmulti_pageapp;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.os.Looper;

public class SplashActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        // This delays the splash screen by 3 seconds before going to OnboardingOneActivity.
        new Handler(Looper.getMainLooper()).postDelayed(() -> {
            Intent intent = new Intent(SplashActivity.this, OnboardingOneActivity.class);
            startActivity(intent);
            finish();
        }, 3000); // This is the time for the splash screen.
    }
}

File: Vian Rechel Academy Incorporated (Multi-page App)/app/src/main/java/com/example/vianrechelacademyincorporatedmulti_pageapp/SplashActivity.kt
import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import androidx.appcompat.app.AppCompatActivity
import com.example.vianrechelacademyincorporatedmulti_pageapp.OnboardingOneActivity
import com.example.vianrechelacademyincorporatedmulti_pageapp.R

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        // This delays the splash for 3 seconds then go to OnboardingOneActivity.
        Handler(Looper.getMainLooper()).postDelayed({
            val intent = Intent(this, OnboardingOneActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000) // This is the time for the splash screen.
    }
}
